src/intercept_util.c

```diff
@@ -575,6 +578,9 @@ print_syscall(char *b, const char *name, unsigned args, ...)
 			long whence = va_arg(ap, long);
 			b += sprintf(b, "%ld (%s)", whence,
 					desc_whence(whence));
+		} else if (format == F_2INT) {
+			int *ints = va_arg(ap, int *);
+			b += sprintf(b, "[%d, %d]", ints[0], ints[1]);
```

Comment:
We could check for NULL, that is cheap to do.

Reply:
Done.